### DISCUSSION TITLE
Strona nie działa

### DISCUSSION THREAD

**Discussion by UniqueTM2** [TomaszGasior/RadioLista-v3#176] [1. Błędy Bugs]
Hej.
Strona RL nie działa, błąd 502.

**Comment by TomaszGasior** [comment]
Obecnie strona działa, natomiast widzę w logach, że błąd 502 był zwracany  kilkukrotnie na przestrzeni całego dnia.

W takich sytuacjach najlepiej odczekać kilkanaście minut i spróbować ponownie. Zazwyczaj problem tego typu nie jest zależny ode mnie i powinien sam ustąpić po odczekaniu krótkiej chwili.

**Comment by UniqueTM2** [comment]
Wiesz, ale to się dzieje kilka razy na dzień, wcześniej tak nie było. Co próbuje wejść na strone to nie działa. Jak się sezon zacznie bedzie to mega denerwujące i uciążliwe

**Reply by TomaszGasior** [reply]
Jestem w kontakcie z dostawcą infrastruktury serwerowej. Niestety poza tym nic nie mogę zrobić. Potwierdzam, że rzeczywiście problem występuje — spotkałem się z błędem 502 przed chwilą, a w logach widać wyraźne nasilenie od 13 lutego.

**Reply by TomaszGasior** [reply]
Według logów z ostatnich dwóch dni błąd przestał już występować.

**Reply by HyperDX** [reply]
> Według logów z ostatnich dwóch dni błąd przestał już występować.

Błąd nadal występuje. Ponownie ten sam komunikat.

**Reply by TomaszGasior** [reply]
Czy błąd występuje jedynie na stronie głównej czy także na podstronach?

**Reply by TomaszGasior** [reply]
U mnie błąd — gdy już go napotykam — wystepuje tylko na stronie głównej. Podstrony (gdy manualnie dopiszę coś po ukośniku, np. /moje-wykazy) działają bez problemu.

**Reply by HyperDX** [reply]
> U mnie błąd — gdy już go napotykam — wystepuje tylko na stronie głównej. Podstrony (gdy manualnie dopiszę coś po ukośniku, np. /moje-wykazy) działają bez problemu.

A widzisz, tego akurat nie zdążyłem sprawdzić. Parę minut temu strona główna zaczęła ładować się bez problemu :).

**Reply by TomaszGasior** [reply]
Ten błąd ma dwie niepokojące cechy: występuje nieregularnie i nie mam nad nim kompletnie żadnej kontroli…

**Reply by TomaszGasior** [reply]
@HyperDX Spróbuj sprawdzić następnym razem, gdy błąd wystąpi, czy np. po dopisaniu `/logowanie`, `/moje-wykazy`, `/cokolwiek` w adresie strony coś się pokaże, czy też będzie bład.

**Comment by TomaszGasior** [comment]
Dzisiaj udało mi się potwierdzić i częściowo zdiagnozować usterkę. Błąd wciąż istnieje i dotyczy prawie wyłącznie strony głównej, ale obecnie jest z jakiegoś powodu „schowany”. https://github.com/TomaszGasior/RadioLista-v3/issues/212
